<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Index</class>
 <widget class="QMainWindow" name="Index">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>354</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Index</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,5,1">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="myimages.qrc">
          <normaloff>:/image/images/4g.png</normaloff>:/image/images/4g.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="myimages.qrc">
          <normaloff>:/image/images/wireless.svg</normaloff>:/image/images/wireless.svg</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="myimages.qrc">
          <normaloff>:/image/images/bl.svg</normaloff>:/image/images/bl.svg</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="myimages.qrc">
          <normaloff>:/image/images/locate.svg</normaloff>:/image/images/locate.svg</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit_2"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(171, 237, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Sunken</enum>
                 </property>
                 <property name="text">
                  <string>称重传感器状况：</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="weight_label">
                 <property name="text">
                  <string>320</string>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>  kg</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="sizeConstraint">
                <enum>QLayout::SetNoConstraint</enum>
               </property>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>去皮</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>稳定</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>零位</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>自动过磅</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0">
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>皮重</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>货号</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="mylineedit" name="lineEdit_carId">
               <property name="text">
                <string>浙B8888</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="mylineedit1" name="lineEdit_artId">
               <property name="text">
                <string>水果</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>车号</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>备注</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEdit_remark">
               <property name="text">
                <string>羽流商贩</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
           <item>
            <widget class="myWidget" name="shipin" native="true">
             <widget class="QLabel" name="iplabel">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>270</width>
                <height>96</height>
               </rect>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGraphicsView" name="graphicsView_2"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="menu">
        <property name="font">
         <font>
          <family>Bitstream Charter</family>
         </font>
        </property>
        <property name="text">
         <string>菜单</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="yiciguobang">
        <property name="text">
         <string>一次过磅</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="erciguobang">
        <property name="text">
         <string>二次过磅</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>去皮</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>置零</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="QRcode">
        <property name="text">
         <string>二维码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="print">
        <property name="text">
         <string>打印</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>mylineedit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>mylineedit1</class>
   <extends>QLineEdit</extends>
   <header>mylineedit1.h</header>
  </customwidget>
  <customwidget>
   <class>myWidget</class>
   <extends>QWidget</extends>
   <header>mywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myimages.qrc"/>
 </resources>
 <connections/>
</ui>
